src/SolverInterface/variables.jl

```diff
@@ -25,10 +25,15 @@ isvalid(m::AbstractMathProgModel, ref::VariableReference) = throw(MethodError())
 
 """
     delete!(m::AbstractMathProgModel, ref::VariableReference)
-
+    
 Delete the referenced variable from the model.
+
+    delete!(m::AbstractMathProgModel, refs::Vector{VariableReference})
+
+Delete the referenced variables in the vector `refs` from the model.
 """
 Base.delete!(m::AbstractMathProgModel, ref::VariableReference) = throw(MethodError())
+Base.delete!(m::AbstractMathProgModel, refs::Vector{VariableReference}) = throw(MethodError())
```

Comment:
The fallback should probably be
`Base.delete!(m::AbstractMathProgModel, refs::Vector{VariableReference}) = delete!.(m, refs)`